keanu-python/keanu/algorithm/sampling.py

```diff
@@ -32,6 +32,16 @@ def get_sampler(self) -> JavaObject:
 
 
 class MetropolisHastingsSampler(PosteriorSamplingAlgorithm):
+    """
+    :param proposal_distribution: The proposal distribution for metropolis hastings. Options are 'gaussian' and 'prior'.
```

Comment:
nit: Can Metropolis Hastings have each first letter capitalised as it's a name of an algorithm.